AllJoyn/Platform/Bridge/BridgeCore/Bridge.vcxproj

```diff
@@ -288,8 +289,10 @@
       <Optimization>MaxSpeed</Optimization>
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <IntrinsicFunctions>true</IntrinsicFunctions>
-      <PreprocessorDefinitions>QCC_OS_GROUP_WINDOWS;NDEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>BRIDGE_WIN10;QCC_OS_GROUP_WINDOWS;_SCL_SECURE_NO_WARNINGS;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Release|x86 and Release|arm still don't build.

---
In reply to: [75177875](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75177875) [](ancestors = 75177875,75174093)